packages/slate/src/changes/at-current-range.js

```diff
@@ -73,6 +73,40 @@ Changes.addMark = (change, mark) => {
 }
 
 /**
+ * Add a list of `marks` to the characters in the current selection.
+ *
+ * @param {Change} change
+ * @param {Mark} mark
+ */
+
+Changes.addMarks = (change, marks) => {
```

Comment:
Could this method's logic be implemented like the `addMarksAtRange` one you've added where it simply loops `marks.forEach( ... )` and calls into the old `addMark`? That way we don't end up with two almost-identical implementations to keep in sync?

Reply:
done.

Reply:
sry for misunderstanding. Didn't read proper.

Reply:
now locally i have no failing tests but circleci has the new tests all failing with:  
Error: No hyperscript creator found for tag: "state"
Do you have a idea what is the problem?

Reply:
@ch1ll0ut1 Thank you! If you rebase against master you should probably see the error. In the newest versions of Slate, the `state` has been renamed to `value`.

Reply:
fixed